common-rinkeby.env

```diff
@@ -2,3 +2,5 @@ STABLEX_CONTRACT_ADDRESS=0x4eAfD2b1A2264228B87e35b77e50AA5b9bcC72dc
 ETHEREUM_NODE_URL=https://rinkeby.infura.io/v3/9408f47dedf04716a03ef994182cf150
 NETWORK_NAME=rinkeby
 NETWORK_ID=4
+PRIVATE_KEY=4f3edf983ac636a65a842ce7c78d9aa706d3b113bce9c46f30d7d21715b23b1d
```

Comment:
Is this line intended?

Reply:
Ah no, it seems to be not needed, since it is already defined in the common.env.